README.md

```diff
@@ -263,36 +263,44 @@ Distill steps to setup the matrix server from <https://github.com/spantaleev/mat
 
 ### master
 
+Finished [Display user's nickname and avatar in group #66](https://github.com/wechaty/matrix-appservice-wechaty/issues/66) and [Support image and file messages #86](https://github.com/wechaty/matrix-appservice-wechaty/issues/86)
+
+1. Support the display of the nickname and name of the contact
+2. Support image and file messages
+3. Other details:
+   1. [fix a deprecated function warning](https://github.com/wechaty/matrix-appservice-wechaty/pull/82/commits/88b1d64cf53018a537397defd596f55447e9080a)
+   2. [avoid inviting your own wechaty acount to matrix room](https://github.com/wechaty/matrix-appservice-wechaty/pull/83/commits/ad3aebed0de4bb129f7c43523225c3065d321526)
+
 ### v0.8 Mar 1, 2020
 
 Finished  [#13](https://github.com/Chatie/matrix-appservice-wechaty/issues/13): [TypeScript] Add typing system to `matrix-js-sdk` and `matrix-appservice-bridge`.
 
 1. Code Refactoring
-1. Add `!login`, `!logout` command with help message
-1. Tested with iPad API([wechaty-puppet-padplus](https://github.com/wechaty/wechaty-puppet-padplus))
+2. Add `!login`, `!logout` command with help message
```

Comment:
We should always keep the original style when contributing to an existing project.

Please use `1.` instead of `2.` and `3.`

Reply:
OK! but I have deleted the 545641826:update-readme branch because I am curious about what will happen. I cannot reopen the PR right now. 😂